From 50034fc47782f774f2bbe3a26813f21eb42bf68e Mon Sep 17 00:00:00 2001
From: Timo Reichl <thetredev@gmail.com>
Date: Fri, 2 May 2025 20:31:27 +0000
Subject: [PATCH] AMBuildScript: add C/C++ defines and flags to fix build issue

Signed-off-by: Timo Reichl <thetredev@gmail.com>
---
 AMBuildScript | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/AMBuildScript b/AMBuildScript
index 2c8712c..a83d852 100644
--- a/AMBuildScript
+++ b/AMBuildScript
@@ -114,6 +114,8 @@ class MMSConfig(object):
         '_vsnprintf=vsnprintf',
         'HAVE_STDINT_H',
         'GNUC',
+        'POSIX',
+        'LINUX',
       ]
       cxx.cflags += [
         '-pipe',
@@ -128,6 +130,9 @@ class MMSConfig(object):
         '-Wno-dangling-else',
         '-msse',
         '-fPIC',
+        # added these to make build possible in 2025
+        '-Wno-error=unknown-pragmas',
+        '-Wno-error=format-truncation',
       ]
 
       cxx.cxxflags += ['-std=c++17']
@@ -141,6 +146,9 @@ class MMSConfig(object):
         '-Wno-non-virtual-dtor',
         '-Wno-overloaded-virtual',
         '-Wno-register',
+        # added these to make build possible in 2025
+        '-Wno-error=unknown-pragmas',
+        '-Wno-error=format-truncation',
       ]
       if (cxx.version >= 'gcc-4.7' or cxx.family == 'clang'):
         cxx.cxxflags += ['-Wno-delete-non-virtual-dtor']
-- 
2.49.0

