From 1572c9b8a3a11d9d8cec64a4457af046c501a6f6 Mon Sep 17 00:00:00 2001
From: Timo Reichl <thetredev@gmail.com>
Date: Mon, 21 Apr 2025 00:47:33 +0200
Subject: [PATCH] loader/utility/mm_FindPattern: don't use register prefix for
 loop variable

Signed-off-by: Timo Reichl <thetredev@gmail.com>
---
 loader/utility.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/loader/utility.cpp b/loader/utility.cpp
index 5de2716..70006c2 100644
--- a/loader/utility.cpp
+++ b/loader/utility.cpp
@@ -562,7 +562,7 @@ void *mm_FindPattern(const void *libPtr, const char *pattern, size_t len)
 	while (ptr < end)
 	{
 		found = true;
-		for (register size_t i = 0; i < len; i++)
+		for (size_t i = 0; i < len; i++)
 		{
 			if (pattern[i] != '\x2A' && pattern[i] != ptr[i])
 			{
-- 
2.39.5

