From 423b4ed1d18fe241dd0b74e649557fa73e71c3cc Mon Sep 17 00:00:00 2001
From: Timo Reichl <thetredev@gmail.com>
Date: Mon, 21 Apr 2025 01:00:04 +0200
Subject: [PATCH] public/tier0/platform.h: fix dangling-else compile error

Signed-off-by: Timo Reichl <thetredev@gmail.com>
---
 public/tier0/platform.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/public/tier0/platform.h b/public/tier0/platform.h
index b0f9c18b..ffd182ac 100644
--- a/public/tier0/platform.h
+++ b/public/tier0/platform.h
@@ -1875,7 +1875,7 @@ template < size_t NUM, class T > struct AlignedByteArray_t : public AlignedByteA
 	struct ALIGN_N( ALIGN ) AlignedByteArrayExplicit_t< NUM, T, ALIGN > \
 	{ \
 		/* NOTE: verify alignment in the constructor (which may be wrong if this is heap-allocated, for ALIGN > MEMALLOC_MAX_AUTO_ALIGN) */ \
-		AlignedByteArrayExplicit_t()	{ if ( (ALIGN-1) & (size_t)this ) DebuggerBreakIfDebugging(); } \
+		AlignedByteArrayExplicit_t()	{ if ( (ALIGN-1) & (size_t)this ) { DebuggerBreakIfDebugging(); } } \
 		T *			Base( void )		{ ValidateAlignmentExplicit<T,ALIGN>(); return (T *)&m_Data; } \
 		const T *	Base( void ) const	{ ValidateAlignmentExplicit<T,ALIGN>(); return (const T *)&m_Data; } \
 	private: \
-- 
2.39.5

